utils/network.go

```diff
@@ -57,6 +57,18 @@ func DoNetworking(args *skel.CmdArgs, conf NetConf, result *current.Result, logg
 			return err
 		}
 
+		mac, err := net.ParseMAC("EE:EE:EE:EE:EE:EE")
+		if err != nil {
+			return fmt.Errorf("failed to parse MAC Address: %v", err)
```

Comment:
Hm, wondering if we should return an error in these cases, or just _log_ an error.

thought process is - we don't do this today and it works on most systems, should we just fall back to today's behavior if for some reason we fail to set the mac on a given system?

Reply:
@tmjd @fasaxc WDYT?

Reply:
Makes sense to me.  Just log an error and only do LinkSetHardwareAddr if it is successful.

Reply:
Yeah, and if `LinkSetHardwareAddr` fails then we carry on as normal (but log an error).

Reply:
Sounds fine to me.  (TBH, I'd be tempted to panic if parsing a fixed MAC fails.  That should work 100% of the time so the most likely scenario for that to fail is if we accidentally introduced a typo into the MAC during maintenance.)